flow/activities/flowable.go

```diff
@@ -460,8 +458,7 @@ func (a *FlowableActivity) SetupQRepMetadataTables(ctx context.Context, config *
 	}
 	defer connectors.CloseConnector(ctx, conn)
 
-	err = conn.SetupQRepMetadataTables(ctx, config)
-	if err != nil {
+	if err := conn.SetupQRepMetadataTables(ctx, config); err != nil {
```

Comment:
```suggestion
	if err = conn.SetupQRepMetadataTables(ctx, config); err != nil {
```
don't think we need this to be scoped, wherever applicable